dist/stat_data.py

```diff
@@ -874,6 +874,8 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_prev_skip_ge_100', 'cursor prev calls that skip greater than or equal to 100 entries'),
     CursorStat('cursor_prev_skip_lt_100', 'cursor prev calls that skip less than 100 entries'),
     CursorStat('cursor_prev_skip_total', 'Total number of entries skipped by cursor prev calls'),
+    CursorStat('cursor_reposition', 'Total number of times the cursor tried to reposition itself'),
+    CursorStat('cursor_reposition_failed', 'Total number of times cursor fails to reposition'),
```

Comment:
Should we just crash if we fail? Why we need this stats then?

Reply:
We don't crash all the time, e.g. `WT_ROLLBACK`.

Reply:
You updated the success one, but not the fail one. Was that deliberate?

Reply:
My bad! I have updated the description [8b60e67](https://github.com/wiredtiger/wiredtiger/pull/7687/commits/8b60e6798b306234876f69b9e02b8db3cba889d1). Let me know if that sounds right.